pragmatic/fields.py

```diff
@@ -11,7 +11,6 @@
 from django.utils.encoding import smart_str
 from django.utils.translation import gettext_lazy as _
 
-from django_filters.constants import EMPTY_VALUES
```

Comment:
I think this import is missing now, it should be replaced by `from django.core.validators import EMPTY_VALUES`?